server/dialect-idms/src/main/java/org/eclipse/lsp/cobol/dialects/idms/IdmsDialectVisitor.java

```diff
@@ -103,6 +106,40 @@ public List<IdmsCopybookDescriptor> visitStartRule(IdmsParser.StartRuleContext c
     return result;
   }
 
+  @Override
+  public List<IdmsCopybookDescriptor> visitObtainStatement(IdmsParser.ObtainStatementContext ctx) {
```

Comment:
As far as I can tell, there are other commands (those marked with LRF - ERASE, MODIFY, (obtain), STORE) that support the same syntax
https://techdocs.broadcom.com/us/en/ca-mainframe-software/database-management/ca-idms-reference/19-0/dml-reference-for-cobol/cobol-data-manipulation-language-dml-statements/erase-lrf-cobol.html

Reply:
@VitGottwald can you help? Do we need to check all this commands if they produce IF statement?

Reply:
Absolutely. The client only reported the one they ran into, but if there are other similar cases, we should fix them as well.

It might be best to check with the IDMS team to verify the list from Slavek and check if it is complete and to provide a sample compile for all of them.